web/moduly/finance.php

```diff
@@ -63,24 +63,30 @@
 
 <div style="clear:both"></div>
 
-<h2 id="placeni">Platba</h2>
 <?php if(!$zaplaceno){ ?>
   <p>
   <?php if($u->stat()=='CZ'){ ?>
-    <strong>Číslo účtu:</strong> <?=UCET_CZ?>
+    <h2 id="placeni">Platba</h2>
```

Comment:
Úpravy v tomto souboru na rychlý pohled vypadají ok, jen je to obecně bolehlav (těch podmínek už je tam moc, za což nemůžeš a nějak to postupně narostlo) a stálo by to (spíš do budoucna zmiňuju) dát do šablony. Hodím na to během týdne ještě druhé oko, ale spíš to už bude OK.